protos/sources/vega/checkpoint/v1/checkpoint.proto

```diff
@@ -224,3 +226,45 @@ message AssetAction {
   bool erc20_bridge_stopped = 11;
   bool erc20_bridge_resumed = 12;
 }
+
+message ELSShare {
+  // Party ID of the liquidity provider.
+  string party_id = 1;
+  // The equity-like share of the liquidity provider, representing their share in the total provided liquidity expressed as a value between 0 and 1.
+  string share = 2;
+  // Amount committed. This is the total commitment made. It does not necessarily match the LP balance.
+  string supplied_stake = 3;
+  // LP stake, weighted according to the time the stake was committed and market growth.
+  string virtual_stake = 4;
+  // Average entry valuation. As time increases the weight of stake, this is the overall valuation of the current stake.
+  string avg = 5;
```

Comment:
maybe we can have a longer name for that ?
`average_entry_valuation` gives me more to chew than just `avg`.

Reply:
I agree with this, 'avg' is pretty sparse

Reply:
It's checkpoint data, this isn't really something that is part of an API in any way shape or form